code/obj/item/clothing/helmets.dm

```diff
@@ -506,16 +506,16 @@
 
 	setupProperties()
 		..()
-		setProperty("meleeprot_head", 2)
+		setProperty("meleeprot_head", 1)
 		setProperty("disorient_resist_eye", 100)
 
 	proc/flip_down()
-		c_flags |= (COVERSEYES | UNCHOKE)
-		setProperty("meleeprot_head", 2)
+		src.c_flags |= (COVERSEYES | BLOCKCHOKE)
+		setProperty("meleeprot_head", 1)
 		setProperty("disorient_resist_eye", 100)
 
 	proc/flip_up()
-		c_flags &= ~(COVERSEYES | UNCHOKE)
+		src.c_flags &= SPACEWEAR
```

Comment:
```suggestion
		src.c_flags &= ~(COVERSEYES | BLOCKCHOKE)
```

Reply:
woundnt that block choking when its flipped up? referencing http://byondref.qtqt.cf/#/operator/&=
it says something like 
A &= B
If A and B are lists, items in A that are not in B are removed. 
witch is the purpose of flip up here?

Reply:
im probably wrong, but my gut is telling me im right from how testing various ways to do this went

Reply:
I was wrong lol